packages/aurora-ether/src/bridged-ether/sendToEthereum/index.ts

```diff
@@ -246,6 +246,8 @@ export async function recover (
   sender: string = 'todo',
   options?: TransferOptions & {
     nep141Factory?: string
+    etherNep141Factory?: string
+    etherNep141FactoryMigrationHeight?: number
```

Comment:
TransferOptions already contains this args

Reply:
https://github.com/aurora-is-near/rainbow-bridge-client/pull/98/commits/a49efb543ac2aa0ea430a994b940bdf50181e4cd